index.js

```diff
@@ -2015,6 +2015,23 @@
     return new RegExp(source, flags);
   });
 
+  //# regexEscape :: String -> String
+  //.
+  //. Takes a string which may contain regular expression metacharacters,
+  //. and returns a string with those metacharacters escaped.
+  //.
+  //. Properties:
+  //.
+  //.   - ∀ `s` ∈ String: `S.test(S.regex('', S.regexEscape(s)), s)`
```

Comment:
Looks good to me, though I wonder whether `∀ s ∈ String` may be a little too foreign for others. I'd also suggest still including `=` in the property statement, despite `S.test` already returning a `Boolean` value.

e.g:
```
forall s :: String. S.test(S.regex('', S.regexEscape(s)), s) = true
```
It would be interesting to see how much effort would be involved in parsing these comments to be executed as actual tests with JSVerify.